userguide/system.rst

```diff
@@ -1046,7 +1050,7 @@ Currently available alert services:
 * `VictorOps <https://victorops.com/>`__
 
 
-.. warning:: These alert services might use a third party commercial
+.. warning:: These alert services may use a third party commercial
```

Comment:
"may" usually implies permission ("yes, you may do that").  "might" is about possibility, and is the right one to use here.